### PR TITLE
fixup! soundwire: intel_auxdevice: add kernel parameter for mclk divider

### PR DESCRIPTION
**Pull request by bardliao**
Add description of the kernel parameter in in kernel-parameters.txt as pre Vinod's suggestion.

### CODE REVIEW

File: Documentation/admin-guide/kernel-parameters.txt [hunk 1]
```diff
@@ -5989,6 +5989,10 @@
 			non-zero "wait" parameter.  See weight_single
 			and weight_many.
 
+	sdw_mclk_divider=[SDW]
+			Specify the MCLK divider for Intel SoundWire buses in
+			case the BIOS does not provide the clock rate properly.
+
```
Comment (plbossart) [inline Documentation/admin-guide/kernel-parameters.txt:5995]: this makes no sense to me. We have tons of kernel parameters that are for developers only. We don't want to maintain or advertise them as a supported configuration interface.

**Comment by plbossart** [comment]
from Documentation/admin-guide/kernel-parameters.rst

"
This document may not be entirely up to date and comprehensive. The command
"modinfo -p ${modulename}" shows a current list of all parameters of a loadable
module. Loadable modules, after being loaded into the running kernel, also
reveal their parameters in /sys/module/${modulename}/parameters/. Some of these
parameters may be changed at runtime by the command
``echo -n ${value} > /sys/module/${modulename}/parameters/${parm}``.
"

I think modinfo is more than enough here.

**Comment by bardliao** [comment]
Allright, let's close the PR.

**Comment by bardliao** [comment]
Reopen as the maintainer is requesting to add document in kernel-parameters.txt

**Review by lgirdwood — APPROVED** [review]

**Comment by bardliao** [comment]
submitted to upstream. Close for now

**Pull request closed without merging**